onnxruntime/python/tools/transformers/fusion_options.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, model_type):
 
         if model_type == "clip":
             self.enable_embed_layer_norm = False
+        self.model_impl = "hf"
```

Comment:
By adding the fusion changes into `fusion_bart_attention.py`, I don't think we need this.